app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -1,12 +1,14 @@
 import React, { useState } from "react";
 
 import { Formik, Form } from "formik";
-import { Button, Pane } from "neetoui/v2";
+import { Button, Pane, Select } from "neetoui/v2";
 import { Input, Textarea } from "neetoui/v2/formik";
 
 import notesApi from "apis/notes";
 import formValidationSchemas from "constants/formValidationSchemas";
 
+import constants from "../constants";
+
 export default function NoteForm({ onClose, refetch, note, isEdit }) {
```

Comment:
Please rename the component in a better manner. Also, we don't need to keep it inside a folder called `Pane`.